packages/synapse-react-client/src/components/SynapseHomepage.tsx

```diff
@@ -80,17 +96,29 @@ export const SynapseHomepage: React.FunctionComponent<SynapseHomepageProps> = ({
                       'noopener',
                     )
                   }
-                  variant="primary-900"
+                  variant="contained"
+                  color="darkPrimary"
```

Comment:
Similarly added a new "darkPrimary" color to MUI theme to handle the dark buttons on the homepage